app/static/src/scss/footer.scss

```diff
@@ -1,57 +1,68 @@
+/* SCSS for Footer */
 
-/* CSS for Footer */
+@mixin on-mobile{
+   @media screen and (width <= 640px){
+       @content;
+   }
+}
 
 
 .govuk-footer  {
-    padding-bottom: 40px;
- }
-
- .govuk-footer__tna-logo {
-    width: 100px;
- }
-
- .govuk-footer__meta {
-    margin: 0;
-    align-items: center;
-    gap: 40px;
- }
-
- .govuk-footer__meta-item {
-    display: block;
-    margin: 0;
- }
-
- .govuk-footer__inline-list-item {
-    margin-bottom: 10px;
- }
-
- @media screen and (width <= 640px) {
-    .govuk-footer__meta {
-        flex-direction: column;
-     }
-
-     .govuk-footer__link {
-        order: 2;
-     }
-
-     .govuk-footer__meta-item--grow {
-        flex-basis: auto;
-     }
-
-     .govuk-footer__inline-list-item  {
-        display:block;
-        margin-right: 0;
-        margin-bottom: 0;
-        text-align:center;
-        padding: 10px 0;
-     }
-
-     .govuk-footer__licence-logo{
-        margin:0 auto 10px;
-        display:block;
-     }
-
-     .govuk-footer__licence-description {
-        text-align: center;
-     }
- }
+   padding-bottom: 40px;
+
+
+&__tna-logo {
+   width: 100px;
+}
+
+&__meta {
+   margin: 0;
+   align-items: center;
+   gap: 40px;
+@include on-mobile {
+flex-direction: column;
+}
+}
+
+&__meta-item {
+   display: block;
+   margin: 0;
+}
+
+&__inline-list-item {
+   margin-bottom: 10px;
+@include on-mobile {
+       display:block;
+       margin-right: 0;
+       margin-bottom: 0;
+       text-align: center;
+       padding: 10px 0;
+}
+}
+
+&__link {
+@include on-mobile {
```

Comment:
is this pattern of

```
&__<subclass> {
@include on-mobile {
some_property: value
```

just reducing duplication so that the mobile styling is included on all components? or is this something else?

If so, does that mean we need to always have the 
`@include on-mobile {` line in every scss definition we make?

Reply:
clarified on call - reducing duplication